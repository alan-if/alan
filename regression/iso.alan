location едц≈ƒ÷
end location.

start at едц≈ƒ÷.
