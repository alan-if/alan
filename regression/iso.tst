alan iso.alan
arun iso < iso.input
