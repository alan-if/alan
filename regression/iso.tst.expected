########## iso ##########
../compiler/alan iso
../interpreter/arun -n iso < iso.input
########## /bin/sh  iso.tst ##########

        No warnings or errors detected.
        2 informational message(s).



Едцедц

> 

Do you want to RESTART, RESTORE or QUIT ? 
